apps/gateway/lib/gateway/router.ex

```diff
@@ -27,6 +27,8 @@ defmodule Gateway.Router do
     get "/auth/:provider/token/:token_id", Controllers.AuthController, :validate_token
 
     put "/users/:user_id", Controllers.UserController, :update
+    get "/users/:user_id/claim_daily_reward", Controllers.UserController, :claim_daily_reward
+    get "/users/:user_id/get_daily_reward_status", Controllers.UserController, :get_daily_reward_status
```

Comment:
I think these routes should be under the "curse" scope since these are operations that belongs there

Reply:
Alright. Done.